app/internal/whatslink.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
```

Comment:
I think it would be easier to understand if the name of the file would be "whatsapp" :)